front/components/SubscriptionContactUsDrawer.tsx

```diff
@@ -30,7 +30,7 @@ export function SubscriptionContactUsDrawer({
       const formURL = `https://docs.google.com/forms/d/e/1FAIpQLSdZdNPHm0J1k5SoKAoDdmnFZCzVDHUKnDE3MVM_1ii2fLrp8w/viewform?usp=pp_url&entry.1203449999=${encodeURIComponent(
         email
       )}`;
-      ClientSideTracking.trackClickEnterpriseContactUs({ email }).finally(
+      void ClientSideTracking.trackClickEnterpriseContactUs({ email }).finally(
```

Comment:
why do we need a void here? The finally should make the linter happy, isn't?

Reply:
It does not seen as a dangling promise

Reply:
Strange, probably related to the new versions of everything.